test-clients/src/main/java/com/hedera/services/yahcli/README.md

```diff
@@ -0,0 +1,95 @@
+# Yahcli v0.0.1
+A command line interface that is able to perform the following actions against a specified network.
+
+1. Account Operations
+    1. Get Balance.
+2. System File Operations
+    1. Download all/specific file.
+    2. Upload a System file.
+3. Fees Operations
+    1. List Basic Transaction and Query fees of all/specific service.
+
+## Getting Started
+### Setting up your environment
+
+Create a folder structure like this.<br>
+![folderStructure](folderStructure.png)
+
+The _config.yml_ should have contents like:
```

Comment:
Can you point to a complete config.yml file you are using or provide a separate one along the README.md?